testsuite/MDAnalysisTests/analysis/test_leaflet.py

```diff
@@ -42,3 +44,11 @@ def test_leaflet_finder(self):
         assert_equal(bottom_heads.indices, np.arange(2521,4670,12), err_msg="Found wrong leaflet lipids")
 
 
+    def test_string_vs_atomgroup_proper(self):
+        from MDAnalysis.analysis.leaflet import LeafletFinder
```

Comment:
I was simply copying the in-method import strategy already in place for this unit testing module, which I assumed was justified if the code was already using that approach. The testing module does attempt to skip if `scipy` is not available, which probably explains it. I thought about placing the import statement in the `setUp` method, but then I figured there must be some justification for not having done that in the first place.